source/fundamentals/authentication/mechanisms.txt

```diff
@@ -281,11 +281,16 @@ Pass the location of your client certificate file as the value of
 .. literalinclude:: /code-snippets/authentication/x509.js
    :language: javascript
 
-TLS/SSL Options
-~~~~~~~~~~~~~~~
+.. tip::
+   
+   To learn more about enabling TLS on a connection, see the
+   :ref:`fundamentals page <node-connect-tls>`.
```

Comment:
Suggestion: 
I think using the title of the page for the link title would be more descriptive than "fundamentals page".

E.g.
"To learn more about enabling TLS on a connect, see :ref\`\<node-connect-tls\>\`."